charts/nginx-proxy/Chart.yaml

```diff
@@ -1,24 +1,6 @@
 apiVersion: v2
-name: nginx-proxy
+appVersion: 1.16.0
 description: nginx-proxy is a nginx caching proxy in front of repo.jenkins-ci.org
-
-# A chart can be either an 'application' or a 'library' chart.
```

Comment:
Not sure why these comments are removed by udpatecli while those in the other yaml file aren't.

Reply:
it's the updatecli's YAML parser. As soon as you use a target of type YAML or type helmchart (both uses the parser yaml v2), then the processed file looses its comments and has its keys sorted.